installer/roles/image_push/tasks/main.yml

```diff
@@ -9,9 +9,6 @@
   delegate_to: localhost
 
 - name: Remove local images to ensure proper push behavior
-  # TODO: this code will not be necessary if and when docker_image can be configured to push if the image
-  # Already exists locally
-  # Pull request: https://github.com/ansible/ansible/pull/31863
```

Comment:
This PR never merged and this always causes confusion. Making this default to true.